CMakeLists.txt

```diff
@@ -103,6 +107,29 @@ if("${_compiler_arch}" STREQUAL "")
     set(_compiler_arch ${CMAKE_SYSTEM_PROCESSOR})
 endif()
 
+# LLVM component setup
+set(LLVM_COMPONENTS core demangle objcarcopts orcjit)
+
+if(${_compiler_arch} MATCHES "x64")
```

Comment:
will this cause any issues for cross-compilation?

Reply:
I'll verify this when I get the tests working again.